phylogenetic/defaults/hmpxv1/config.yaml

```diff
@@ -24,51 +24,52 @@ filter:
 ### Set 1: Non-B.1 sequences: use all
 ### Set 2: B.1 sequences: small sample across year/country, maybe month
 subsample:
-  non_b1:
-    group_by: "--group-by lineage year country"
-    sequences_per_group: "--sequences-per-group 50"
-    other_filters: "outbreak!=hMPXV-1 clade!=IIb"
-    exclude_lineages:
-      - B.1
-      - B.1.1
-      - B.1.2
-      - B.1.3
-      - C.1
-      - C.1.1
-      - E.1
-      - E.2
-      - E.3
-      - B.1.4
-      - B.1.5
-      - B.1.6
-      - D.1
-      - B.1.7
-      - B.1.8
-      - B.1.9
-      - B.1.10
-      - B.1.11
-      - B.1.12
-      - B.1.13
-      - B.1.14
-      - B.1.15
-      - B.1.16
-      - B.1.17
-      - B.1.18
-      - B.1.19
-      - B.1.20
-      - F.1
-      - F.2
-      - F.3
-      - F.4
-      - F.5
-      - F.6
-      - B.1.21
-      - B.1.22
-      - B.1.23
-  b1:
-    group_by: "--group-by country year"
-    sequences_per_group: "--subsample-max-sequences 300"
-    other_filters: "--exclude-where outbreak!=hMPXV-1 clade!=IIb"
+  non_b1: >-
+    --group-by lineage year country
+    --sequences-per-group 50
+    --exclude-where
+    outbreak!=hMPXV-1
+    clade!=IIb
```

Comment:
[minor] can you increase the indentation of the values following `--exclude-where`, or does the mixed indentation not work with YAML folded blocks?

Reply:
In YAML folded blocks, any extra indentation adds a newline.

Reply:
Bummer!